### ISSUE TITLE
Unlock requirements

### ISSUE THREAD

**Issue by uSpike** [ansible/molecule#1612]
Molecule is using `pbr` and `requirements.txt` to set its dependencies, which are locked to specific versions.  This makes it difficult to install other python packages on top of molecule that may depend on the same packages.  For example, I want to incorporate a library that depends on a newer version of `testinfra`, however molecule depends on exactly `testinfra==1.16.0`, so my only option is to open a PR to ask molecule developers to bump the version of `testinfra`.

Would the maintainers be open to changing to using more lax version requirements in `requirements.txt` or switching to `setuptools` ?

**Comment by webknjaz** [comment]
Fair enough. Actually I'm a big fan of switching to setuptools. I'll probably do smth like https://github.com/ansible/ansible-lint/pull/389 here as well.

**Comment by lubo** [comment]
@gundalow Not really. #1607 simply states that Molecule does not use the latest release of ansible-lint. This issue states that it's inconvenient when a package, that's usually being used as part of a larger project, locks the dependencies to particular versions. I agree with @uSpike and I'd like to add that this can easily cause dependency hell and lots of headaches.
